### PR TITLE
Dev1 branch

### PR DESCRIPTION
**Pull request by Vyke030422**

### CODE REVIEW

File: canlineapp/lib/main.dart [hunk 1]
```diff
@@ -11,7 +13,7 @@ class MainApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return const MaterialApp(
-      home: SplashScreen(
+      home: Infohub(
```
Comment (CapillanesFrizty) [inline canlineapp/lib/main.dart:16]: Change this to scaffold layout

File: canlineapp/lib/screen/homescreen/infohub.dart [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                        CardFinancial(onTap: () => handleCardTap('Financial', 3)),
+                        CardBlogs(onTap: () => handleCardTap('Blogs', 2)),
+                        CardClinic(onTap: () => handleCardTap('Clinics', 0)),
+                      ],
+                    ),
+            ),
+          ],
+        ),
+      ),
+    );
```
Comment (CapillanesFrizty) [inline canlineapp/lib/screen/homescreen/infohub.dart:90]: Inside the Body Encapsulate it to 1 Widget

**Review by CapillanesFrizty — CHANGES_REQUESTED** [review]
Please change those stuff, I'm looking forward to the changes, Hope you made it, Happy Coding💪🎗️

File: canlineapp/lib/main.dart (hunk 1 above)
Comment (Vyke030422) [reply canlineapp/lib/main.dart]: Done

**Review by CapillanesFrizty — APPROVED** [review]

**Pull request merged**